assets/js/app.js

```diff
@@ -9,6 +13,12 @@ require('../css/superfish.css');
 require('../css/pelagos-module.css');
 require('../css/messages.css');
 
+const routes = require('../../public/js/fos_js_routes.json');
+import Routing from '../../vendor/friendsofsymfony/jsrouting-bundle/Resources/public/js/router.min.js';
+
+Routing.setRoutingData(routes);
+global.Routing = Routing;
```

Comment:
@mickel1138 : Made this a global variable, so that we can use Routing variable in public/js files and we dont need those old fos_js_routing script statements in our html index files

Reply:
It's not working on a real server. The 'pelagos-symfony' disappears.